packages/next/src/build/utils.ts

```diff
@@ -1173,6 +1178,13 @@ export type AppConfig = {
   dynamic?: AppConfigDynamic
   fetchCache?: 'force-cache' | 'only-cache'
   preferredRegion?: string
+
+  /**
+   * When true, the page will be served using partial prerendering.
+   * This setting will only take affect if it's enabled via
+   * the `experimental.ppr = "incremental"` option.
+   */
+  experimental_ppr?: boolean
```

Comment:
This is the only snake case property, can we do `experimentalPPR` instead?

Reply:
As we will eventually deprecate it, having it ugly like this keeps it consistent with the existing `unstable_cache` and similar API's that are not designed to be permanant.